policyengine_us/variables/gov/states/mt/tax/income/mt_capital_gains_tax_indiv.py

```diff
@@ -0,0 +1,88 @@
+from policyengine_us.model_api import *
+
+
+class mt_capital_gains_tax_indiv(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana net long-term capital gains tax when married couples file separately"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/12/Form_2_2023_Instructions.pdf#page=6"  # Net Long-Term Capital Gains Tax Table
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
```

Comment:
we might need to use the threshold.yaml here and create a new lower rate, because the threshold is reduced by the nonqualified income which we do not represent now

Reply:
for the new lower rate, do we need to change it to one scale parameter per filing status like below, or just put all filing status breakdown in one parameter file?
![image](https://github.com/PolicyEngine/policyengine-us/assets/97819588/ede4ec8d-b014-475c-8dd0-1d5597a64dd5)

Reply:
We might need individual parameter files but if you think you can use a scale parameter to make it more concise then feel free to do that

Reply:
Got it.